apm-agent-plugins/apm-logging-plugin/apm-log4j2-plugin/src/test/java/co/elastic/apm/agent/log4j2/Log4j2InstrumentationTest.java

```diff
@@ -102,7 +109,15 @@ public void close() {
 
         @Override
         public String getLogFilePath() {
-            return ((RandomAccessFileAppender) ((org.apache.logging.log4j.core.Logger) log4j2Logger).getAppenders().get("FILE")).getFileName();
+            assertThat(log4j2Logger)
+                .isInstanceOf(org.apache.logging.log4j.core.Logger.class);
+            org.apache.logging.log4j.core.Logger logger = (org.apache.logging.log4j.core.Logger) log4j2Logger;
+
+            assertThat(logger.getAppenders()).containsKey("FILE");
+            Appender fileAppender = logger.getAppenders().get("FILE");
+            assertThat(fileAppender).isInstanceOf(RandomAccessFileAppender.class);
+
+            return ((RandomAccessFileAppender) fileAppender).getFileName();
```

Comment:
[for reviewer] this does not help fixing the tests, but would help to identify test flakyness on windows